code/datums/mutations/_mutations.dm

```diff
@@ -93,7 +93,7 @@
 	return
 
 /datum/mutation/human/proc/on_losing(mob/living/carbon/human/owner)
-	if(owner && istype(owner) && (owner.dna.mutations.Remove(src)))
+	if(!owner || !istype(owner) || !(owner.dna.mutations.Remove(src)))
```

Comment:
what broke mutations is that i added an early return for readability WITHOUT reversing this if check to TRUE on fail. in the other pr, they return TRUE on a completed on_losing when TRUE is supposed to be for failed removals.